oop.ipynb

```diff
@@ -598,9 +551,11 @@
     "hidden": true
    },
    "source": [
-    "We can also define classes in terms of *other* classes, in which case the new classes **inherit** the attributes and methods from the classes in terms of which they're defined.\n",
+    "It is common for a class to have magic methods. These are identifiable by the \"dunder\" (i.e. **d**ouble **under**score) prefixes and suffixes, such as `__init__()`. These methods will get called **automatically** as a result of a different call, as we'll see below.\n",
+    "\n",
+    "For more on these \"magic methods\", see [here](https://docs.python.org/3/reference/datamodel.html#special-method-names).\n",
```

Comment:
@gadamico suggested having the prior link about magic methods too (https://www.geeksforgeeks.org/dunder-magic-methods-python/)

Reply:
See commit: https://github.com/flatiron-school/ds-object_oriented_programming_2/pull/1/commits/4b24841bf63e9ff4a95d5cd4e5143b30521d6e31